worldengine/draw.py

```diff
@@ -161,14 +161,47 @@ def __setitem__(self, item, value):
         self.pixels[item] = value
 
 
-def draw_simple_elevation(data, width, height, sea_level, target):
+def draw_simple_elevation(world, sea_level, target):
     """ This function can be used on a generic canvas (either an image to save
         on disk or a canvas part of a GUI)
     """
-    for y in range(height):
-        for x in range(width):
-            e = data[y][x]#TODO: numpy-notation [y,x] is a little faster; all passed arrays would have to be numpy-arrays
-            r, g, b = elevation_color(e, sea_level)
+    min_elev_sea = None
+    max_elev_sea = None
+    min_elev_land = None
+    max_elev_land = None
+    for y in range(world.height):
```

Comment:
I had been working on most of the image-drawing methods in  https://github.com/Mindwerks/worldengine/pull/141 . I won't be finished first, so I'll have to adapt to your changes. But: Please remember that now all arrays are numpy-arrays and this part of the code could be drastically shortened/sped up.
You can find some (deactivated) code here, should you need an example:
https://github.com/Mindwerks/worldengine/pull/141/files#diff-c209aa2ca5a09dd35a85ec7844837b6eR230

That does pretty much the same as you are doing here.